cmake/modules/SetUpLinux.cmake

```diff
@@ -108,16 +108,10 @@ if(CMAKE_COMPILER_IS_GNUCXX)
   # Select flags.
   set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O3 -g -DNDEBUG"  CACHE STRING "Flags for release build with debug info")
   set(CMAKE_CXX_FLAGS_RELEASE        "-O3 -DNDEBUG"     CACHE STRING "Flags for release build")
-  set(CMAKE_CXX_FLAGS_OPTIMIZED      "-Ofast -DNDEBUG"  CACHE STRING "Flags for a highly optimised build")
   set(CMAKE_CXX_FLAGS_DEBUG          "-g"               CACHE STRING "Flags for a debug build")
-  set(CMAKE_CXX_FLAGS_DEBUGFULL      "-g3"              CACHE STRING "Flags for a debug build with more debug info")
-  set(CMAKE_CXX_FLAGS_PROFILE        "-g3 -ftest-coverage -fprofile-arcs" CACHE STRING "")
```

Comment:
The original idea of this flags was to (I guess 'eventually') have it know/set the compiler depend profiling options.